src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/helpdoc.txt

```diff
@@ -0,0 +1,630 @@
+    --authentication-type         : Type of authentication to use with the VM. Allowed values are
+                                    password, ssh.  Default: password.
+    --generate-ssh-keys           : Generate SSH public and private key files if missing.
+    --ssh-key                     : The SSH public key or public key file path. Use --generate-ssh-
+                                    keys to regen ssh keys.
+
+Network Arguments
+    --disallow-public-ip-address  : To allow public ip address set to true or false. If omitted,
+                                    based on the defaulted subnet this will be set to true or false.
+    --subnet                      : Name of the subnet to reference an existing one in lab. If
+                                    omitted, lab's existing one subnet will be selected
+                                    automatically.
+    --vnet-name                   : Name of the virtual network to reference an existing one in lab.
+                                    If omitted, lab's existing one VNet and subnet will be selected
+                                    automatically.
+
+Global Arguments
+    --debug                       : Increase logging verbosity to show all debug logs.
+    --help -h                     : Show this help message and exit.
+    --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                    Default: json.
+    --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                    information and examples.
+    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm delete -h
+
+Command
+    az lab vm delete: Delete virtual machine.
+        This operation can take a while to complete.
+
+Arguments
+    --lab-name          [Required]: The name of the lab.
+    --name              [Required]: The name of the virtual machine.
+    --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                    using 'az configure --defaults group=<name>'.
+
+Global Arguments
+    --debug                       : Increase logging verbosity to show all debug logs.
+    --help -h                     : Show this help message and exit.
+    --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                    Default: json.
+    --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                    information and examples.
+    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm list -h
+
+Command
+    az lab vm list: Command to retrieve my vms from the Azure DevTest Lab.
```

Comment:
As we talked, I've updated the command to default to my vms and added flag --all for getting all vms. Here is the updated docs

```
(env) vishrut@visshamac azure-cli (ipconfig) $ az lab vm list -h

Command
    az lab vm list: Command to retrieve my vms from the Azure DevTest Lab.

Arguments
    --lab-name          [Required]: Name of the Lab.
    --resource-group -g [Required]: Name of lab's resource group.
    --expand                      : Specify the expand query.
    --object-id                   : Owner's object id. If omitted, we'll pick one if available.
    --order-by                    : The ordering expression for the results using OData notation.
    --top                         : The maximum number of resources to return from the operation.

Filter Arguments
    --all                         : List all virtual machines in the lab. Cannot be used with
                                    --filters.
    --claimable                   : List of claimable virtual machines in the lab. Cannot be used
                                    with --filters.
    --filters                     : The filter to apply on the operation.
```